openshift/release/knative-eventing-kafka-channel-ci.yaml

```diff
@@ -120,6 +121,16 @@ rules:
     resources:
       - rolebindings
     verbs: *everything
+  - apiGroups:
+      - rbac.authorization.k8s.io
+    resources:
+      - clusterrolebindings
+    verbs: *everything
+  - apiGroups:
+      - monitoring.coreos.com
+    resources:
+      - servicemonitors
+    verbs: *everything
   - apiGroups:
       - "coordination.k8s.io"
```

Comment:
this file is generated, by running 

```shell
make RELEASE=v0.24.3 generate-release
```

on the root folder. I'd vote AGAINST manually updating this file, but instead patching the actual yamls 

One thing to keep in mind is, that often in the past we had success patching our RBAC changes directly on upstream yamls... which removes the need to do manual tweaks/patches on our forks

Reply:
@skonto see my comments above

Reply:
/hold

Reply:
@matzew you mean patch the yamls on the release branch for openshift-knative/eventing?

> One thing to keep in mind is, that often in the past we had success patching our RBAC changes directly on upstream yamls... which removes the need to do manual tweaks/patches on our forks

I could try.

Reply:
> I could try but these are Prometheus specific and so it seems to me irrelevant upstream and add would add noise for a specific type of monitoring system.

ok. lets' than get a patch that is applied every time we creatte a new branch, e.g. like: https://github.com/openshift-knative/eventing-kafka/tree/main/openshift/patches